crates/fuzzing/src/oracles/engine.rs

```diff
@@ -54,13 +54,16 @@ pub fn choose(
         }
     }
 
+    if engines.is_empty() {
+        return Ok(None);
+    }
```

Comment:
Slightly surprised we would ever have a case where we can't run the test case in Wasmtime. What is even the point at that time?

Reply:
I think the issue this is addressing is if testing Wasmtime against Wasmtime has been disabled using the `ALLOWED_ENGINES` environment variable.

Reply:
Ah yeah this is only applicable when you're doing something like `ALLOWED_ENGINES=wasmi` locally where I was trying to only differentially-execute against `wasmi`